### PR TITLE
Allow case where user id is type UUIDField

### LINKED ISSUE #51: NoReverseMatch when viewing drip timeline if user uses UUID
My users have a UUID, and django-drip-campaigns doesn't account for that possibility in the URL reverse.

`Traceback (most recent call last)
File "/usr/local/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py", line 76, in __call__
return self.application(environ, start_response)
File "/usr/local/lib/python3.8/site-packages/django/core/handlers/wsgi.py", line 133, in __call__
response = self.get_response(request)
File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 128, in get_response
response = self._middleware_chain(request)
File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner
response = response_for_exception(request, exc)
File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 103, in response_for_exception
response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 138, in handle_uncaught_exception
return debug.technical_500_response(request, *exc_info)
File "/usr/local/lib/python3.8/site-packages/django_extensions/management/technical_response.py", line 40, in null_technical_500_response
raise exc_value.with_traceback(tb)
File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
response = get_response(request)
File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 179, in _get_response
response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/usr/local/lib/python3.8/contextlib.py", line 75, in inner
return func(*args, **kwds)
File "/usr/local/lib/python3.8/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
response = view_func(request, *args, **kwargs)
File "/usr/local/lib/python3.8/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
response = view_func(request, *args, **kwargs)
File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 233, in inner
return view(request, *args, **kwargs)
File "/usr/local/lib/python3.8/site-packages/drip/admin.py", line 65, in timeline
return render(request, 'drip/timeline.html', locals())
File "/usr/local/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
content = loader.render_to_string(template_name, context, request, using=using)
File "/usr/local/lib/python3.8/site-packages/django/template/loader.py", line 62, in render_to_string
return template.render(context, request)
File "/usr/local/lib/python3.8/site-packages/django/template/backends/django.py", line 61, in render
return self.template.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 170, in render
return self._render(context)
File "/usr/local/lib/python3.8/site-packages/django/test/utils.py", line 96, in instrumented_test_render
return self.nodelist.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
bit = node.render_annotated(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
return compiled_parent._render(context)
File "/usr/local/lib/python3.8/site-packages/django/test/utils.py", line 96, in instrumented_test_render
return self.nodelist.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
bit = node.render_annotated(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
return compiled_parent._render(context)
File "/usr/local/lib/python3.8/site-packages/django/test/utils.py", line 96, in instrumented_test_render
return self.nodelist.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
bit = node.render_annotated(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 62, in render
result = block.nodelist.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
bit = node.render_annotated(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 211, in render
nodelist.append(node.render_annotated(context))
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 312, in render
return nodelist.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
bit = node.render_annotated(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 211, in render
nodelist.append(node.render_annotated(context))
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 312, in render
return nodelist.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
bit = node.render_annotated(context)
File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 446, in render
url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
File "/usr/local/lib/python3.8/site-packages/django/urls/base.py", line 87, in reverse
return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
File "/usr/local/lib/python3.8/site-packages/django/urls/resolvers.py", line 685, in _reverse_with_prefix
raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'view_drip_email' with arguments '(1, 4, 7, UUID('7ab86abc-7856-4f91-be1b-8bd6147b836c'))' not found. 1 pattern(s) tried: ['admin/drip/drip/(?P<drip_id>[0-9]+)/timeline/(?P<into_past>[0-9]+)/(?P<into_future>[0-9]+)/(?P<user_id>[0-9]+)/$']`

### PR DESCRIPTION
**Pull request by OmenApps**
Fixes #51

### CODE REVIEW

File: drip/admin.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 from django import forms
 from django.contrib import admin
+from django.db.models import UUIDField
```
Comment (nsantos16) [inline drip/admin.py]: Are you using this import? I think you only check the type of ID with `get_internal_type` function, if that is the case please remove it

Comment (OmenApps) [reply drip/admin.py]: Corrected this.

File: drip/admin.py [hunk 2]
```diff
@@ -148,3 +165,4 @@ class SentDripAdmin(admin.ModelAdmin):
 
 
 admin.site.register(SentDrip, SentDripAdmin)
+
```
Comment (nsantos16) [inline drip/admin.py]: In order to follow the flake8 style guide of the project, please remove this blank line at end of the file

File: drip/admin.py [hunk 3]
```diff
@@ -129,13 +131,28 @@ def get_urls(self):
...
+            ]
+        else:
+            my_urls += [
+                path(
+                    '<int:drip_id>/timeline/<int:into_past>/'
+                    '<int:into_future>/<int:user_id>/',
+                    self.av(self.view_drip_email),
+                    name='view_drip_email'
+                ),
+            ]
```
Comment (nsantos16) [inline drip/admin.py:152]: Nice 👏

**Comment by nsantos16** [comment]
Hey @OmenApps thanks for your contribution! 🙌
We encourage you to keep contributing to this project.

**Comment by OmenApps** [comment]
Thanks Nicolas. I saw this package pop up on PyPI yesterday, and it was just what we needed in one of our projects. Will be glad to keep contributing where we can.

**Review by nsantos16 — APPROVED** [review]

**Review by kaozdl — APPROVED** [review]

**Comment by kaozdl** [comment]
@OmenApps thank you for contributing!
can you add a test for the other branch of the if?
Overall looks good to go

**Review by jiss2891 — APPROVED** [review]

**Comment by OmenApps** [comment]
@kaozdl, will work on this next week. May have some questions about recommended approach.

**Comment by kaozdl** [comment]
Sure, feel free to ask.
If you want to schedule a call i might be available

**Comment by brunomichetti** [comment]
Hello @OmenApps, thanks for contributing!
Were you able to add tests to this PR? We would like to merge, but with some automatic testing first

**Comment by brunomichetti** [comment]
Hello @OmenApps, this PR will be closed because another PR was merged. This other PR is the same feature that you implemented but adding automatic testing.
Thank you for contributing!

**Pull request closed without merging**